Cargo.toml

```diff
@@ -287,7 +287,7 @@ lru = { version = "0.11.0", default-features = false, optional = true }
 maxminddb = { version = "0.23.0", default-features = false, optional = true }
 md-5 = { version = "0.10", default-features = false, optional = true }
 mongodb = { version = "2.6.0", default-features = false, features = ["tokio-runtime"], optional = true }
-nats = { version = "0.24.0", default-features = false, optional = true }
+async-nats = { git = "https://github.com/nats-io/nats.rs.git", rev = "a59b21eabceef45736f6f7ca987987e8c3edb805", default-features = false, optional = true }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is there a reason why we need a specific `rev`?

Reply:
I was hoping on 0.32.0 being out soon. I've changed to 0.31.0 so that we can go forward with merging and dependabot can manage to update to 0.32